1_setup_my_env_as_root.ubuntu.sh

```
@@ -0,0 +1,55 @@
+  && sudo apt-get clean
+
+echo "📦 Installing Paho MQTT C library..."
+git clone --depth=1 https://github.com/eclipse/paho.mqtt.c.git /tmp/paho.mqtt.c
+cd /tmp/paho.mqtt.c
+cmake -Bbuild -S. \
+  -DPAHO_BUILD_STATIC=TRUE \
+  -DPAHO_BUILD_SHARED=TRUE \
+  -DPAHO_WITH_SSL=TRUE \
+  -DPAHO_BUILD_CMAKE_EXPORTS=ON \
+  -DCMAKE_INSTALL_PREFIX=/usr
+cmake --build build --target install -- -j$(nproc)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)